cfgov/preprocessed/css/organisms/content-sidebar.less

```diff
@@ -0,0 +1,40 @@
+/* topdoc
+  name: Content Sidebar
+  family: cfgov-organism
+  patterns:
+    - name: Default example
+      markup: |
+          <aside class="o-content-sidebar">
+              <*></*>
+          </aside>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-content-sidebar
+  tags:
+    - cfgov-molecules
+*/
+
+.o-content-sidebar {
+
+    .respond-to-range(@tablet-min, @tablet-max {
```

Comment:
So if there is only one or if the last one is even then it will be full width. I'm not sure it's the best ideal now that I think about it.